example/ios/Podfile.lock

```diff
@@ -6,36 +6,36 @@ PODS:
   - MTBBarcodeScanner (5.0.8)
   - permission_handler (3.3.0):
     - Flutter
-  - square_reader_sdk (2.0.0):
+  - square_reader_sdk (2.1.0):
```

Comment:
Why was this bumped to 2.1?

Reply:
We have increased the version [here](https://github.com/square/reader-sdk-flutter-plugin/blob/2a70b0ed0836a389c39ded2e94e56041c180af86/ios/square_reader_sdk.podspec#L6) in square_reader_sdk.podspec, hence it takes version from there.